src/panels/config/network/dialog-ip-detail.ts

```diff
@@ -41,20 +41,49 @@ class DialogIPDetail extends LitElement {
         @closed=${this.closeDialog}
         scrimClickAction
         escapeKeyAction
-        .heading=${createCloseHeading(this.hass, "IP Information")}
+        .heading=${createCloseHeading(
+          this.hass,
+          this.hass.localize("ui.dialogs.dialog-ip-detail.ip_information")
+        )}
       >
         ${ipv4
           ? html`
               <div>
-                <h3>IPv4</h3>
+                <h3>
+                  ${this.hass.localize("ui.dialogs.dialog-ip-detail.ipv4")}
+                </h3>
                 ${ipv4.address
-                  ? html`<div>IP Address: ${ipv4.address?.join(", ")}</div>`
+                  ? html`<div>
+                      ${this.hass.localize(
+                        "ui.dialogs.dialog-ip-detail.ip_address"
+                      )}:
+                      ${ipv4.address?.join(", ")}
```

Comment:
Pass the IP addresses as a param to the translation string:
```suggestion
                      ${this.hass.localize(
                        "ui.dialogs.dialog-ip-detail.ip_address", {address: ipv4.address?.join(", ")}
                      )}
```